features/services/active_docs.feature

```diff
@@ -0,0 +1,26 @@
+Feature: ActiveDocs pages
+  As a provider
+  I want to manage my ActiveDocs of a specific service
+
+  Background:
+    Given a provider is logged in
+      And the provider has 1 active doc
+
+  Scenario: Index renders with the proper elements
+    When I go to the service active docs page
+     And the table should not contain the API
```

Comment:
It would be good to see it as a table probably though a bit too much but explicit.
Or doing as a Percy that we approve as it is design mostly

Reply:
The whole table would be testing much more than what it actually means to test here. And too much more information than desired or needed.

> Or better doing as a Percy that we approve as it is design mostly

Who knows about percy? @thomasmaas ?

Reply:
See https://github.com/3scale/porta/tree/master/features/percy
Not much but if they are important they should go there. Like design bugs we encounter

Reply:
Ok so I don't think it is the case 🤔